CodeEdit/AppDelegate.swift

```diff
@@ -170,13 +165,25 @@ final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject {
     }
 
     @IBAction func openAbout(_ sender: Any) {
+        if tryFocusWindow(of: AboutView.self) { return }
+
         AboutView().showWindow(width: 530, height: 220)
     }
 
     @IBAction func openFeedback(_ sender: Any) {
+        if tryFocusWindow(of: FeedbackView.self) { return }
+
         FeedbackView().showWindow()
     }
 
+    private func tryFocusWindow<T: View>(of type: T.Type) -> Bool {
```

Comment:
Please add documentation to newly added methods.

Reply:
done